Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            //main
+            Interview();
+        }
+
+        public static void Interview()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The Cyclomatic Complexity of this method is 11 which is greater than 10 authorized.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4876897)